solidity/contracts/NeedsUpdateGuard.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.19;
+
+import {BaseGuard} from 'safe-contracts/base/GuardManager.sol';
+import {Enum} from 'safe-contracts/common/Enum.sol';
+import {IVerifierModule} from 'interfaces/IVerifierModule.sol';
+
+/**
+ * @title NeedsUpdateGuard
+ * @notice This guard should prevent the safe from executing any transaction if an update is needed.
+ * @notice An update is needed based on the safe owner's security settings.
+ */
+contract NeedsUpdateGuard is BaseGuard {
+  /**
+   * @notice Emits when the owner changes the update security settings
+   * @param _safe The address of the safe
+   * @param _newSecuritySettings The new security settings, how many seconds the safe trusts the last update
+   */
+  event SecuritySettingsChanged(address indexed _safe, uint256 _newSecuritySettings);
```

Comment:
Same with this event, as the actual security settings arent changing, only the time till we force an update is